CharacterCreation/01_01_Races_Human

```diff
@@ -0,0 +1,145 @@
+accurate throwing. Not all humans practice this technique, but you are one who 
+has mastered this seemingly simple aspect of your anatomy.
+[Negates the aiming action necessary to throw accurately]
+
+Stubborn: 
+All Humans have a hardiness to them that cannot always be explained by logic.
+Some humans are just plain hard to kill, for no other apparent reason than they 
+just refuse to die. 
+[Roll Advantage on Death Saving Throws]
+
+
+==============================
+Human Sub-Species
+==============================
+
+===== Terrans =====
+
+Size: Medium
+Home System: Keplar
+
+Appearance: 
+Heavily resembling the historic humans, there is little variation from even 20th 
+century humans in this pool. The Terrans left Sol behind in the 24th Century, 
+forming a new colony in the Kepler system. 
+
+
+== Available Traits ==
+
+Very Normal: 
+Blending in has always come early to you. People tend to trust you. You seem to 
+belong no matter where you find yourself. 
+[Advantage on Persuade & Deceive ]
+
+Sixth Sense:
+Colloquially often known as the ‘gut’ instinct, sixth sense is that feeling that 
+many Terrans get when they feel something is off. There are different levels of 
+Sixth Sense, and many function in different ways. Some Terrans get goosebumps, 
+while others experience a hair-raising feeling; it can be very hard to get anything 
+past Humans with this sense tuned. 
+[Grants you an extra sense. DM’s discretion for application]
+
+
+
+===== Eastern Earthlings =====
+
+Size: Small/Medium
+Home System: Sol
+
+Inhabiting the areas of Earth east of equator, these humans have adapted to the humid,
+swampy heat of their half of the planet. Small and lean, these humans rarely reach over 
+5’ in height, and are much lighter than their Terran counterparts. As the new earth 
+conditions have stabilized, these humans have been beginning to rebuild some sense of 
+agriculture, but only recently have they ceased their nomadic patterns.
+
+== Available Traits ==
+
+Endurance: 
+Generations of Nomadic behavior have made moving long distances easy for those of 
+Eastern Earthling descent. 
+[First 2 Actions Sprinting are free of penalty. 1 Recovery Action required
+for every 2 Sprinting Actions]
+
+
+
+===== Western Earthlings =====
+
+Size: Large
+Home System: Sol
+
+Living as close to the equator as they dare, these humans are confined to the 
+western edge of the geostorm center. Some have spread further west, but most do 
+not like the absence of sunlight. As their climate tends to be cold to freezing, 
+these Earthlings are larger overall than their Terran counterparts.
+
+[Low-Light Vision]
+
+== Available Traits ==
+
+Quick Constriction: 
+Living in an area ravaged by lighting storms, these humans have developed rapid 
+pupil constriction and dilation. This increases their resistance to not only 
+lightning blasts, but to flash bangs. 
+[Effects from flash-bangs (and related implements) halved]
```

Comment:
Flash-bang is hyphenated elsewhere in the rules. 140 is consistent but 139 has a nonhyphenated spelling.
Nonblocking.